src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -89,6 +105,29 @@ protected void processFiltered(File file, List<String> lines) throws CheckstyleE
         }
     }
 
+    private boolean checkBuildPropertiesFile(File file, String property, String value) throws CheckstyleException {
+        if (!isEmpty(file)) {
+            try {
+                IBuild buildPropertiesFile = parseBuildProperties(file);
+                IBuildEntry binIncludes = buildPropertiesFile.getEntry(property);
+
+                if (binIncludes != null) {
+                    if (binIncludes.contains(value)) {
+                        return true;
+                    }
+                } else {
+                    logger.warn("{} doesn't contain {} property", BUILD_PROPERTIES_FILE_NAME, property);
```

Comment:
All these log statements don't end up in the report wouldn't that make more sense?